packages/components/src/core/ContentCard/index.tsx

```diff
@@ -156,19 +166,19 @@ const ContentCard = React.forwardRef<HTMLDivElement, ContentCardProps>(
         <StyledCardContent
           boundingBox={boundingBox}
           clickableCard={clickableCard}
-          className={classes?.cardContent}
+          className={cardContent}
         >
           <ContentCardTitle
             overlineText={overlineText}
             titleText={titleText}
             subtitleText={subtitleText}
             metadataText={metadataText}
             classes={{
-              cardHeader: classes?.cardHeader,
-              metadataText: classes?.cardMetadata,
-              overlineText: classes?.cardOverline,
-              subtitleText: classes?.cardSubtitle,
-              titleText: classes?.cardTitle,
+              cardHeader: cardHeader,
+              metadataText: cardMetadata,
+              overlineText: cardOverline,
+              subtitleText: cardSubtitle,
+              titleText: cardTitle,
```

Comment:
Super nit! This makes me realize the keys and API are different. E.g., `subtitleText` vs. `cardSubtitle`.

Would it makes sense to just use one, so we have a consistent key to use across component boundaries?

As a side benefit, we can then update the `classes` to be just:
```ts
{
  cardHeader, cardMetadata,  cardOverline, ...
}
```